lib/quoters/WebhookQuoter.ts

```diff
@@ -119,68 +132,71 @@ export class WebhookQuoter implements Quoter {
-            response: hookResponse.data,
-            responseStatus: hookResponse.status,
+        await this.firehose.sendAnalyticsEvent({
+          eventType: AnalyticsEventType.WEBHOOK_RESPONSE,
+          eventProperties: {
+            ...requestContext,
+            ...rawResponse,
+            responseType: WebhookResponseType.NON_QUOTE,
           },
-          `Webhook elected not to quote: ${endpoint}`
-        );
+        });
         return null;
       }
 
       // RFQ provider response failed validation
       if (validation.error) {
         metric.putMetric(Metric.RFQ_FAIL_VALIDATION, 1, MetricLoggerUnit.Count);
         metric.putMetric(metricContext(Metric.RFQ_FAIL_VALIDATION, name), 1, MetricLoggerUnit.Count);
-        this.log.error(
-          {
-            error: validation.error?.details,
-            response,
-            webhookUrl: endpoint,
+        await this.firehose.sendAnalyticsEvent({
+          eventType: AnalyticsEventType.WEBHOOK_RESPONSE,
+          eventProperties: {
+            ...requestContext,
+            ...rawResponse,
+            responseType: WebhookResponseType.VALIDATION_ERROR,
+            validationError: validation.error?.details,
           },
-          `Webhook Response failed validation. Webhook: ${endpoint}.`
-        );
+        });
         return null;
       }
 
       if (response.requestId !== request.requestId) {
         metric.putMetric(Metric.RFQ_FAIL_REQUEST_MATCH, 1, MetricLoggerUnit.Count);
         metric.putMetric(metricContext(Metric.RFQ_FAIL_REQUEST_MATCH, name), 1, MetricLoggerUnit.Count);
-        this.log.error(
-          {
-            requestId: request.requestId,
-            responseRequestId: response.requestId,
+        await this.firehose.sendAnalyticsEvent({
+          eventType: AnalyticsEventType.WEBHOOK_RESPONSE,
+          eventProperties: {
+            ...requestContext,
+            ...rawResponse,
+            responseType: WebhookResponseType.REQUEST_ID_MISMATCH,
+            mismatchedRequestId: response.requestId,
           },
-          `Webhook ResponseId does not match request`
-        );
+        });
         return null;
       }
 
-      const quote = request.type === TradeType.EXACT_INPUT ? response.amountOut : response.amountIn;
-
       metric.putMetric(Metric.RFQ_SUCCESS, 1, MetricLoggerUnit.Count);
       metric.putMetric(metricContext(Metric.RFQ_SUCCESS, name), 1, MetricLoggerUnit.Count);
-      this.log.info(
-        {
-          response: response.toLog(),
-          endpoint: endpoint,
+      await this.firehose.sendAnalyticsEvent({
+        eventType: AnalyticsEventType.WEBHOOK_RESPONSE,
+        eventProperties: {
+          ...requestContext,
+          ...rawResponse,
+          responseType: WebhookResponseType.OK,
         },
-        `WebhookQuoter: request ${
-          request.requestId
-        } for endpoint ${endpoint} successful quote: ${request.amount.toString()} -> ${quote.toString()}}`
-      );
+      });
```

Comment:
Do these firehouse logs show up in CloudWatch? If not, let's keep these info logs because they have been super useful for debugging issues.